bin/tart2ms

```diff
@@ -105,22 +110,28 @@ if __name__ == "__main__":
             "At the moment can only concatenate from JSON or HDF files, not a combination of the two")
 
     if ARGS.json:
-        print("Getting Data from file: {}".format(ARGS.json))
-        print("Writing measurement set '{}'...".format(ARGS.ms))
+        logger.info("Getting Data from file: {}".format(ARGS.json))
+        logger.info("Writing measurement set '{}'...".format(ARGS.ms))
```

Comment:
Generic question. Should we be using the logger for output that the user should always see? I'm happy to approve this but we should discuss.